core/primitives-core/src/version.rs

```diff
@@ -286,6 +287,7 @@ impl ProtocolFeature {
             ProtocolFeature::ExcludeExistingCodeFromWitnessForCodeLen => 148,
             ProtocolFeature::BlockHeightForReceiptId => 149,
             // Place features that are not yet in Nightly below this line.
+            ProtocolFeature::GlobalContracts => 200,
```

Comment:
Why not add it to nightly?

Reply:
I wanted to move it to the nightly separately when I have the complete implementation, to avoid dealing with potential test failures because of unimplemented todos